modopt/opt/algorithms.py

```diff
@@ -480,21 +488,18 @@ class ForwardBackward(SetUp):
     auto_iterate : bool, optional
         Option to automatically begin iterations upon initialisation (default
         is 'True')
-    fista_params: dict,
-        Parameters for the lazy-start and the restarting strategy in FISTA mode
 
     """
 
     def __init__(self, x, grad, prox, cost='auto', beta_param=1.0,
                  lambda_param=1.0, beta_update=None, lambda_update='fista',
                  auto_iterate=True, metric_call_period=5, metrics={},
-                 linear=None, **fista_params):
+                 linear=None, **kwargs):
 
         # Set default algorithm properties
         super(ForwardBackward, self).__init__(
            metric_call_period=metric_call_period,
-           metrics=metrics,
-           linear=linear)
+           metrics=metrics, **kwargs)
```

Comment:
This is going to be a problem because you will have the fista parameters passed in the `SetUp` init. There must be a different way to do this I guess

Reply:
Still testing, will open for review when I am done.

Reply:
Looks OK, now feel free to check.